### PR TITLE
Styled edit input, displayed creation date and created a filter functionality

### PR DESCRIPTION
**Pull request by isabelamuller**

### CODE REVIEW

File: src/Components/CardList/ListType/index.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { IList } from "./types"
+import Card from "../../SingleCard"
+
+const List = ({ type, todoList, setTodoList, completedTodos, setCompletedTodos, nonCompletedTodos, setNonCompletedTodos }: IList) => {
+    const getList = () => {
+        switch (type) {
+            case "allTasks":
+            default:
+                return (
+                    <div>
```
Comment (gustavomdavila) [inline src/Components/CardList/ListType/index.tsx]: [fix] Whenever you're rendering a list you should use either a `ul`, for lists in which the items have no order like this one, or a `ol`, for any list that has some order or hierarchy, like a list of steps in a recipe or a breadcrumb component. Also, you'll need to wrap the `Card` component with a `li` tag.

File: src/Components/Button/styles.ts [hunk 2]
```diff
@@ -14,13 +14,34 @@ export const ButtonStyle = styled.button`
...
-    background-color: #ffdba2;
+    background-color: #facdd4;
     transition-duration: 300ms;
+    color:#ef7c8e;
   }
 `;
 export const ButtonClear = styled(ButtonStyle)`
-  margin-top: 15px;
-  margin-left: 80%;
+  margin-left: 11px;
```
Comment (gustavomdavila) [inline src/Components/Button/styles.ts]: [fix] It's not a good practice to use `px` for margins, font sizes, paddings and so on as it makes them static, this way if the user configures their browser to use a bigger font your application won't be affected. You can use `rem` or `em`. You can install the `sainoba.px-to-rem` extension in vscode and let it convert the `px` values to `rem` automatically.

**Review by gustavomdavila — CHANGES_REQUESTED** [review]
It looks amazing, just a couple tweaks and it'll be good to go.

File: src/Components/Button/styles.ts (hunk 2 above)
Comment (isabelamuller) [reply src/Components/Button/styles.ts]: Thanks! Just updated all of them.

File: src/Components/CardList/ListType/index.tsx (hunk 1 above)
Comment (isabelamuller) [reply src/Components/CardList/ListType/index.tsx]: Just did! Thank you Gustavo.

**Pull request merged**